### DISCUSSION TITLE
How to fund the callStack of set catalog

### DISCUSSION THREAD

**Discussion by peeyee** [dbeaver/dbeaver#34590] [DBeaver Development] [answered]
Hi, guys.
         when trying to set the SQL editor's catalog, but nothing happened, even not receiving a warning/error log message. 

![企业微信截图_17198295736083](https://github.com/dbeaver/dbeaver/assets/47773420/ad1c2793-c0e0-464c-9913-6d55f11a5ab4)
        How to find the callStack of setting catalog? By the way, how to set the logLevel to debug.

**Comment by E1izabeth** [comment, marked as answer]
The class SelectDatabaseDialog creates this dialog.
Here's the stacktrace on double clicking
```
getSelectedObject:295, ObjectListDialog (org.jkiss.dbeaver.ui.navigator.dialogs)
execute:117, SelectActiveSchemaHandler (org.jkiss.dbeaver.ui.actions.datasource)
31 hidden frames
start:292, DBeaverApplication (org.jkiss.dbeaver.ui.app.standalone)
```

**Reply by peeyee** [reply]
Thanks a lot, I can not solve this without your help
